arcade_machine/sprites/label.py

```diff
@@ -14,7 +14,6 @@ def __init__(
         self,
```

Comment:
You can't make changes to the interface of this class without updating all uses of it.

Reply:
Luckily, nothing else was using it when I changed it. So I think we are fine from here on out.